packages/volto/src/components/theme/AppExtras/AppExtras.test.jsx

```diff
@@ -46,6 +46,13 @@ beforeAll(() => {
         <div className="something">{JSON.stringify(props.match)}</div>
       )),
     },
+    {
+      match: '/test-exclude',
```

Comment:
```suggestion
      match: '/frontpage',
      exclude: '/frontpage/images',
      component: jest.fn((props) => (
        <div className="frontpage-content">{JSON.stringify(props.match)}</div>
      )),
    },
```
@dobri1408 you need to redo the test to make it more clear on what your feature does.
You want to show that you can load the content from https://github.com/plone/volto/pull/5621/files#diff-ab6967b7b0acf3c2cb23aea6a5252475fd6c058af52d53b6d1c8fc7a00b1a1b0R53
when the pathname is /test-exclude but when you are inside /text-exclude/edit
then you will get only the content of the match '' everything since that section excludes
the /test-exclude match.

I added a suggestion for you to take into consideration.